pandas/_libs/lib.pyx

```diff
@@ -2315,3 +2312,199 @@ def fast_multiget(dict mapping, ndarray keys, default=np.nan):
             output[i] = default
 
     return maybe_convert_objects(output)
+
```

Comment:
is there a reason you are putting these in lib? why not in parsing.pyx

Reply:
`parsing.pyx` file name does not correspond in meaning to what we do in our functions (there is no parsing in them)